gobblin-service/src/test/java/org/apache/gobblin/service/modules/orchestration/DagManagerTest.java

```diff
@@ -779,15 +785,18 @@ public void testJobKilledSLAMetricsArePerExecutor() throws URISyntaxException, I
         thenReturn(jobStatusIterator2).
         thenReturn(jobStatusIterator3);
 
-    // Run the thread once. All 3 jobs should be emitted an sla exceeded event
+    // Run the thread once. All 3 jobs should be emitted a sla exceeded event
```

Comment:
"an" might be right, https://www.reddit.com/r/ENGLISH/comments/433x3y/do_you_use_a_or_an_before_the_acronym_sla_sla_is/
I will let you decide on this :D

Reply:
Damn today I learned 😅 I will fix that, Intellij's syntax checker flagged it for me.

Reply:
Actually, it should be in uppercase SLA, then intellij recognize the 'As' sound